templates/anbox-cloud/form-data.json

```
@@ -0,0 +1,59 @@
+{
+  "form": {
+    "/anbox-cloud": {
+      "templatePath": "/anbox-cloud/index.html",
+      "isModal": true,
+      "modalId": "anbox-cloud-modal",
+      "formData": {
+        "title": "Let's get in touch",
+        "formId": "3498",
+        "returnUrl": "/anbox-cloud#contact-form-success",
+        "product": ""
+      },
+      "fieldsets": [
+        {
+          "title": "Please briefly tell us about your use case",
+          "id": "comments",
+          "isRequired": false,
+          "fields": [
+            {
+              "type": "long-text",
+              "id": "comments",
+              "placeholder": "Anything you'd like to communicate about your needs or interests?"
+            }
+          ]
+        },
+        {
+          "title": "How should we get in touch?",
+          "id": "about-you",
+          "noCommentsFromLead": true,
+          "fields": [
+            {
+              "type": "text",
+              "id": "company",
+              "label": "Company name",
+              "isRequired": true
+            },
+            {
+              "type": "text",
+              "id": "title",
+              "label": "Job title",
+              "isRequired": true
+            },
+            {
+              "type": "tel",
+              "id": "phone",
+              "label": "Mobile/cell phone number"
+            },
+            {
+              "type": "country",
+              "id": "country",
+              "label": "Country",
+              "isRequired": false
```

```suggestion
              "isRequired": true
```
My mistake, this should be required